### DISCUSSION TITLE
Artifact Hub use for displaying Kyverno policies

### DISCUSSION THREAD

**Discussion by chipzoller** [artifacthub/hub#2968] [Ideas]
The official [Kyverno policy library](https://github.com/kyverno/policies) is by far the largest library of any policy engine for Kubernetes. As a result of Kyverno's very versatile capabilities, there are many different types of policies that can be written. The present number of policies is somewhere around 270 which, at this time, would make Kyverno policies the fourth largest kind on Artifact Hub. In order to help users find the policies which are of interest to them and applicable to their version, we have devised a [custom page on the Kyverno website](https://kyverno.io/policies/) which allows users to locate policies based on a number of filters. Those filters are currently

* Policy type
* Policy category
* (Kyverno) minimum version
* Subject

The filters are built dynamically from the contents of [Kyverno annotations](https://github.com/kyverno/policies/wiki/Kyverno-annotations) which appear on each policy and are read from the policies after conversion to Markdown using a [custom program](https://github.com/kyverno/website/tree/main/render).

In order for the Kyverno project to adopt Artifact Hub as its policy page similar to a project like [Keptn](https://keptn.sh/docs/integrations/), we would like to see a similar set of filters such as those we have built in order to help users narrow down policies. Although Artifact Hub does have [support for most of these Kyverno annotations](https://artifacthub.io/docs/topics/annotations/kyverno/), it does not currently allow them to be used as filter criteria nor as a search filter.

Artifact Hub is a great project and is quite flexible for many different types of artifacts, but its lack of tailoring available to some of the specific needs of those artifacts makes it difficult to adopt as the "official" interface to locating artifacts.

**Comment by tegioz** [comment]
Thanks @chipzoller, we'll look into this 👍
